packages/libs/error-stack/src/lib.rs

```diff
@@ -399,25 +403,26 @@
 //!  Feature   | Description                                                    | implies | default
 //! -----------|----------------------------------------------------------------|---------|--------
 //!  `std`     | Enables support for [`Error`] and, on nightly, [`Backtrace`]   |         | enabled
-//!  `hooks`   |Enables the usage of [`set_display_hook`] and [`set_debug_hook`]| `std`   | disabled
 //! `spantrace`| Enables the capturing of [`SpanTrace`]s                        |         | disabled
 //!  `futures` | Provides a [`FutureExt`] adaptor                               |         | disabled
 //! `small`    | Enable optimizations for the memory footprint of [`Report`]    |         | enabled
 //!  `anyhow`  | Provides conversion from [`anyhow::Error`] to [`Report`]       |         | disabled
 //!   `eyre`   | Provides conversion from [`eyre::Report`] to [`Report`]        |         | disabled
+//!   `glyph`  | Provide color and use of box drawing characters in [`Debug`] output |    | enabled
```

Comment:
Ideas for the feature name @hashintel/error-stack? What about `fancy` or `pretty`? (`glyph` is also fine, just want to raise this at least)

Reply:
`rich_text`?

Reply:
`colorful`?
`color`? 

Maybe for reference: miette uses `fancy`, eyre uses `color` (color-eyre)

Reply:
I don't feel strongly, but do prefer `rich_text` because I think the other two make less sense as stand alone feature flags (rather than as a config option for a logger or something). Like `error-stack -- fancy` is a bit strange to me without knowing _what_ is fancy

Reply:
I think `rich-text` (with `-` instead of `_` if allowed) is fine, even though I'd prefer single-worded feature flags. According to [C-FEATURE](https://rust-lang.github.io/api-guidelines/naming.html#c-feature) we should be fine.

The feature is not only adding colors but also different characters, so `color` does not fit IMO.

Reply:
> error-stack is already fancy 💃

💯

Reply:
My problem with `rich-text` is that it implies that we only do something with the text, but we mainly change colors and use box-drawing characters.

How about `ansi`? Because we use ANSI escape sequences, but then again. `ansi` doesn't *tell* the user what it is about w/o being knowledgable about terminal terminology. The box characters are `semigraphics`.

Names are hard.

Reply:
A cursory google seems to suggest that rich text does imply colours:
> It supports text formatting, such as bold, italics, and underlining, as well as different fonts, font sizes, and colored text.

Otherwise I'd go with something like `fancy-output`, `fancy-print`, or `fancy-debug`, etc. in an attempt to try and be more specific about what we're on about

Reply:
> Names are hard.
One of the hardest parts of software engineering 😁

Reply:
Just in case we want to add more rich text to other sections in the code I think we shouldn't use `fancy-debug`. What about `pretty-print`? At least that's a well known term

Reply:
I really like `pretty-print`!

Reply:
renamed in [fe6e867](https://github.com/hashintel/hash/pull/794/commits/fe6e867fe89e234b4af332cc0038a52a724fdcea)